pytest_motor/plugin.py

```diff
@@ -27,27 +28,26 @@ def event_loop() -> Iterator[asyncio.AbstractEventLoop]:
 
 
 @pytest.fixture(scope='session')
-async def root_directory(pytestconfig: PytestConfig) -> Path:
-    """Return the root path of pytest."""
-    return pytestconfig.rootpath
+async def root_directory() -> Path:
+    """Return the working path."""
+    return Path(tempfile.gettempdir())
```

Comment:
I don't think we want to do this because it will mean that the binary has to be downloaded every run. The reason for using a fixed path is that we can check if the mongod binary has already been downloaded, and then only download it if it doesn't exist.

Reply:
Ok, it's wip. Let me think a better alternative that works on MacOs. There still a few things to fix. I wasnt aware that every push is generating a PR.

Reply:
@AustinScola  just thinking, what if the location is configured via env variable? and defaults to `pytestconfig.rootpath` if not present?

Reply:
Sounds good to me

Reply:
Is the problem that the `pytestconfig.rootpath` doesn't work or is wrong one MacOS?

Reply:
Sorry for the delay, very busy these days.

`pytestconfig.rootpath` value seems to be ok. It starts mongod apparently but then fails to connect the socket. With a modification, I'm pushing you'll be able to override the value with `PYTEST_MOTOR_PATH=/tmp ./scripts/all.sh`.

Reply:
What is the reason for failing to connect to the socket?

Reply:
With your recent changes, it seems to work fine on MacOs. Regarding `PYTEST_MOTOR_PATH ` it could be still useful in some cases. I have a bunch of microservices and downloading the mongo binaries for each of them is cumbersome. Also `readlink -f` is working. You have to install `greadlink` (GNU) and create an alias.